### ISSUE TITLE
MEET WARN:YOU have multiple RING_BUFFER_EXPORT structs in bpf.h. and fail to load in " ./ecli run ..." 

### ISSUE THREAD

**Issue by Wu-Fisher** [eunomia-bpf/eunomia-bpf#62]
When I am trying to use docker to compile one of the examples, I faced the warning:
```
WARN: YOU have multiple RING_BUFFER_EXPORT structs in bpf.h. The first will be used as a export type.
```
and then my run failed because "load failed"

**Comment by Wu-Fisher** [comment]
But I fixed it by changing the "*.h" to "*.bpf.h" and I had no warning with everything passing. 🤣 I wonder why.

**Comment by yunwei37** [comment]
well，maybe I forgot to release the newest docker image in docker hub....Thank you for reporting this😭

The newest version of docker image use xxx.h instead of xxx.bpf.h in examples，for consistent with bcc/libbpf-tools.

**Comment by Wu-Fisher** [comment]
ok🤩, I once change the name because your *.io page's example.  thx u!

**Comment by yunwei37** [comment]
This should be fixed by [release 0.3.0](https://github.com/eunomia-bpf/eunomia-bpf/releases/tag/v0.3.0) now.
